cgi-bin/DW/FeedCanonicalizer.pm

```diff
@@ -49,49 +49,49 @@ sub canonicalize {
 
     my $uri_str = $uri->as_string;
 
-    given ( $uri_str ) {
+    {
         # Let's see if this looks "LJ-ish".
-        when( m!^https?://(?:users|community|syndicated)\.([^/]+)/+([a-z0-9\-_]+)/$LJISH_URL_PART$!i ) {
+        if ( $uri_str =~ m!^https?://(?:users|community|syndicated)\.([^/]+)/+([a-z0-9\-_]+)/$LJISH_URL_PART$!i ) {
             my ( $host, $sub, $feed, $extra, $spare ) = ( $1, $2, $3, $4, $5 );
-            continue if $feed ~~ m/^rss/i && ! $LJISH_SITES{$host};
-            continue if $spare && ! $LJISH_SITES{$host};
-            return make_ljish( $host, $sub, $feed, $orig_uri, $extra );
+            return make_ljish( $host, $sub, $feed, $orig_uri, $extra ) unless 0
```

Comment:
I can happily not do that, I was just trying to have all the "continue bail-out cases" have the "or $someCase" form v.s. the first one just being " $someCase"